MapboxNavigation/CarPlayManager.swift

```diff
@@ -236,6 +241,7 @@ public class CarPlayManager: NSObject, CPSearchTemplateDelegate {
         let dateCreatedAttribute = [MMEEventKeyCreated: timestamp]
         eventsManager.manager.enqueueEvent(withName: MMEventTypeCarplayDisconnect, attributes: dateCreatedAttribute)
         eventsManager.manager.flush()
+        isConnectedToCarPlay = false
```

Comment:
Why not set this property to false in `application(_:didDisconnectCarInterfaceController:from:)` instead of here? If the property is set to false here in `sendCarPlayDisconnectEvent(_:)`, I’d expect it to be set to true in the corresponding `sendCarPlayConnectEvent(_:)` method, not in `application(_:didConnectCarInterfaceController:to:)`.